docs/metadata/configure-audit-logging.md

```diff
@@ -0,0 +1,108 @@
+
+```
+aws configure --> with user access keys
+cd {desired-directory}
+aws s3 sync s3://customer-backend-events-xyz
+```
+
+## Audit events
+
+This table lists events for each Entity/Action along with the Request parameters grouped by the entity.
+
+:::info
+
+Please note, Prophecy Uses GraphQL queries so User might find some difference in Request and Response parameters depending upon where the Queries are used from.
+
+:::
+
+| Entity     | Query                               | Description                                         | Request Parameters                                                                                                   |
+| ---------- | ----------------------------------- | --------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| Fabric     | fabricDetailQuery                   | Get Fabric Details                                  | ["id"]                                                                                                               |
+|            | addFabric                           | Add a Fabric                                        | ["name", "ownerUid"]                                                                                                 |
+|            | updateOwnedBy                       | Update Team owing the Fabric                        | ["id","targetUid"]                                                                                                   |
+|            | userFabricQuery                     | Get all Fabrics for User                            | ["uid"]                                                                                                              |
+| Project    | addProject                          | Add a project                                       | ["name","forkMode","language", "ownerUid", "mainBranchModificationAllowed"]                                          |
+|            | getDetails                          | Get Details of a Project                            | ["projectId"]                                                                                                        |
+|            | Project List Query                  | List all projects for User                          | ["uid"]                                                                                                              |
+|            | teamProjectAvailable                | Available Projects for that Team                    | ["uid", "language"]                                                                                                  |
+|            | add Project Dependency              | Add a dependency Project to Current                 | ["projectId", "DependencyProjectUid"]                                                                                |
```

Comment:
How come some of the queries have spaces and some not? Some are uppercase and some not? Can we clean this up?

Reply:
fixed.